packages/ui/src/lib/listMenu/ListMenuItem.svelte

```diff
@@ -0,0 +1,187 @@
+<script context="module" lang="ts">
+	export interface ListMenuItem {
+		title: string;
+		id: string;
+		url: string;
+		children?: ListMenuItem[];
+		isExpanded?: boolean;
+	}
```

Comment:
It's not ideal that the name of the `ListMenuItem` component is the same as the type of the elements in one of its props. This requires rename the type when importing into `ListMenu`, which then results in seeing `export let items: Item[];` in `ListMenu` but `export let children: ListMenuItem[] = [];` in the `ListMenuItem` component - it isn't imemdiately obvious that these are the same type. I'm not sure of the best name, but `ListMenuEntry` for the type would work.